eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -42,14 +42,17 @@ jobs:
     # Linux musl x64
     - ${{ if eq(parameters.platform, 'Linux_musl_x64') }}:
       - ${{ if eq(parameters.jobParameters.isFullMatrix, false) }}:
-        - (Alpine.312.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200602002622-e06dc59
+        - (Alpine.313.Amd64.Open)ubuntu.1604.amd64.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-helix-amd64-20210910135845-8a6f4f3
```

Comment:
Both branches are the same - we can probably drop the conditions. Even better - only add 3.14 on `isFullMatrix=true` as it's unlikely a PR will see benefit from testing both.

Reply:
Removed 3.13 from isFullMatrix=true.

---
A side note: in some places, we are using this syntax as here. or more generally:
```yaml
- ${{ if eq(someParm, false) }}:
  - foo
- ${{ if eq(someParm, true) }}:
  - bar
```
in others, it is different:

```yaml
- ${{ if eq(someParm, true) }}:
  - foo
- ${{ if ne(someParm, true) }}:
  - bar
```
unless the latter is to cover `true` or  `<everything else>`, it would be nice to define a convention as YAML + this custom expression syntax ain't easy to digest. 😅

Reply:
The thing is in yaml things decay to strings relatively easily. It's safer to do eq true, and ne true.

Reply:
Yup, so latter one is better if values are different.